hadoop-ozone/dist/dev-support/bin/dist-layout-stitching

```diff
@@ -114,6 +114,7 @@ run cp "${ROOT}/hadoop-ozone/objectstore-service/target/hadoop-ozone-objectstore
 cp -r "${ROOT}/hadoop-hdds/docs/target/classes/docs" ./
 
 #Copy docker compose files
-run cp -p -R "${ROOT}/hadoop-ozone/dist/src/main/compose" .
```

Comment:
Yes, it's required. We need filtering as the hadoopfs jar files contain the version number. We started to use the HADOOP_CLASSPATH=.....*.jar (use wildcard instead of exact version) but it didn't work for me. I replaced it to filter the compose folder and copy the filtered version.